tooling/cli/src/interface/rust.rs

```diff
@@ -1208,8 +1208,21 @@ fn tauri_config_to_bundle_settings(
     .unwrap_or(BundleResources::List(Vec::new()));
   #[allow(unused_mut)]
   let mut depends_deb = config.linux.deb.depends.unwrap_or_default();
+  #[allow(unused_mut)]
+  let mut provides_deb = config.linux.deb.provides.unwrap_or_default();
+  #[allow(unused_mut)]
+  let mut conflicts_deb = config.linux.deb.conflicts.unwrap_or_default();
+  #[allow(unused_mut)]
+  let mut replaces_deb = config.linux.deb.replaces.unwrap_or_default();
```

Comment:
`depends_deb` is a special case because we need to modify it but these new variables are not and so should be removed from here and just passed directly below, same for RPM variables